package/yast2-snapper.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Nov 18 11:15:53 UTC 2015 - teclator@gmail.com
+
+- disable delete and modify snapshots if not exist (bsc#951179)
```

Comment:
Nitpicking: Please refer to the buttons here so it's clear that this is mostly a UI change:
"Disable delete and modify buttons if no snapshots exist"